tools/cargo_makepad/src/android/java/dev/makepad/android/Makepad.java

```diff
@@ -36,7 +38,7 @@ interface Callback {
     static native void onMidiDeviceOpened(long cx, String name, Object midi_device, Callback callback);
     static native void onHideTextIME(long cx, Callback callback);
     static native void onResizeTextIME(long cx, int ime_height, Callback callback);
-    static native void copyToClipboard(long cx, Callback callback);
-    static native void pasteFromClipboard(long cx, String content, Callback callback);
-    static native void cutToClipboard(long cx, Callback callback);
+    static native void onCopyToClipboard(long cx, Callback callback);
+    static native void onPasteFromClipboard(long cx, String content, Callback callback);
+    static native void onCutToClipboard(long cx, Callback callback);
```

Comment:
Renaming to not confuse with the actual copyToClipboard operations that originate from rust.